src/components/library_card_table/LibraryCardTable.js

```diff
@@ -12,7 +12,7 @@ import lockIcon from '../../images/lock-icon.png';
 import LibraryCardFormModal from '../modals/LibraryCardFormModal';
 import { LibraryCard } from '../../libs/ajax';
 import ConfirmationModal from '../modals/ConfirmationModal';
-import {Delete, Update} from '@material-ui/icons';
+import {Delete, Update} from '@mui/icons-material';
```

Comment:
There is a non-trivial change to the icon sizes here. New view is in the foreground, dev is in the background.

<img width="1709" alt="Screenshot 2023-02-14 at 8 32 57 AM" src="https://user-images.githubusercontent.com/116679/218754297-44358fb8-1060-4dd4-bc06-d0914638ac35.png">

Reply:
Yeah, you're right. I'll address this and add some tests around the mui stuff today.